src/style.css

```diff
@@ -61,11 +61,15 @@
 
 .names-and-scores {
   max-height: 50vh;
-  border: 2px solid black;
   display: flex;
   flex-direction: column;
   align-items: flex-start;
   padding: 0;
+  overflow-y: auto;
+}
+
+.border {
+  border: 2px solid black;
```

Comment:
- [ ] In comparison to the wireframe given, kindly adjust the `Leaderboard` "logo" to align with the score board. 
- You can see its seperated from your design and does not ideally give a proper visual appeal. Kindly fix this 👍 

## Your work:
![image](https://github.com/devalibello/leaderboard/assets/50990925/17e836e8-c7db-4a59-9c0d-f01f3b4edbe7)

## Wireframe:
![image](https://github.com/devalibello/leaderboard/assets/50990925/e89d7bd8-0743-4eef-8888-ba1a34e57d3c)